packages/react/src/create-strict-api/__tests__/index.test.tsx

```diff
@@ -152,6 +152,35 @@ describe('createStrictAPI()', () => {
 
       expect(getByTestId('div')).toHaveCompiledCss('val', 'ok', { target: ':hover' });
     });
+
+    it('should type error when circumventing the excess property check', () => {
+      const styles = cssMap({
+        primary: {
+          color: 'var(--ds-text)',
+          accentColor: 'red',
+          // NOTE: This should error, but doesn't until the other errors are resolved, only one error at a time
+          asdf: 'red',
+          '&:hover': {
+            color: 'var(--ds-text-hover)',
+            // @ts-expect-error — Type 'string' is not assignable to type 'undefined'.ts(2322)
```

Comment:
These aren't really perfect ways to test this, but I believe this type-checks in `yarn build`, right?